eventarc/testing/audit_storage.e2e_test.go

```diff
@@ -49,6 +66,8 @@ func TestAuditStorageSinkService(t *testing.T) {
 	fmt.Printf("client.Do: %s %s\n", req.Method, req.URL)
 
 	if got := resp.StatusCode; got != http.StatusOK {
+		b, _ := ioutil.ReadAll(resp.Body)
+		fmt.Println(b)
```

Comment:
suggestion: Rather than just print b, should we add a test case around the response body?

Reply:
I added this to give us a better message than "want 200 got 400" in the case of errors.
There is a test for body content in `../audit_storage/main_test.go`, which i think is sufficient.